src/tests/test_latest.py

```diff
@@ -2,11 +2,9 @@
 
 
 def test_is_latest():
-    last_millesime_name = ""
-    for member in Millesime:
-        if member.name > last_millesime_name:
-            last_millesime_name = member.name
+    recent_millesime_name = max((member.name for member in Millesime))
```

Comment:
C'est un détail de syntaxe propre à Python, mais pas besoin de la doubles parenthèses dans le cas d'un appel de fonction.

> if a function call has a single positional argument, it can be a generator expression without extra parentheses, but in all other cases you have to parenthesize it.

[source](https://peps.python.org/pep-0289/#id16)